jest.base.config.js

```diff
@@ -10,8 +11,8 @@ module.exports = {
         '!**/__*__/**', // ignore tests, acceptance, stories, etc
     ],
     transform: {
-        '\\.css\\.ts$': '@vanilla-extract/jest-transform',
-        '^.+\\.(t|j)sx?$': ['@swc/jest', {...swcConfig, sourceMaps: 'inline'}],
+        '\\.css\\.ts$': ['@vanilla-extract/jest-transform', swcTransform],
```

Comment:
For some reason, the swc transform is now required in the `css.ts` files. 
It makes sense but I'm not sure why this was working before.